docs/blueprints/EP033.rst

```diff
@@ -0,0 +1,252 @@
+  3. EVC_C_D:
+    1. User_C --> User_D: User_C (Switch_2 interface 3) to PP2 (Switch_1 interface 4) and PP2 (Switch_1 interface 4) to User_D (Switch_1 interface 2)
+    2. User_D --> User_C: User_D (Switch_1 interface 2) to PP3 (Switch_2 interface 5) and PP3 (Switch_2 interface 5) to User_C (Switch_2 interface 3)
+  4. EVC_B_C:
+    1. User_B --> User_C: User_B (Switch_2 interface 2) to PP3 (Switch_2 interface 5) and PP3 (Switch_2 interface 5) to User_C (Switch_2 interface 3)
+    2. User_C --> User_B: User_C (Switch_2 interface 3) to PP3 (Switch_2 interface 5) and PP3 (Switch_2 interface 5) to User_B (Switch_2 interface 2)
+
+
+Pros of this solution:
+  * All EVCs will be enhanced with telemetry.
+  * User_A doesn't share bandwidth with User_D and User_E on Switch_1 although User_A, User_D and User_E are on the same switch.
+
+Cons of this solution:
+  * Same limitations of the previous example for the intra-switch EVCs.
+  * There are two **proxy ports** on Switch_1.
+
+
+
+  3. **Example 3: Intra-switch and Inter-switch EVCs with no proxy port and a dedicated proxy port**
+
+For this example, User_A and User_C won't have a **proxy port** and User_B will have a dedicated **proxy port**.
+
+Consider EVCs previously created by the `mef_eline` napp:
+
+  * EVC_A_B, connecting User_A (Switch_1 interface 2) to User_B (Switch_3 interface 1)
+  * EVC_A_C, connecting User_A (Switch_1 interface 2) to User_C (Switch_3 interface 3)
+  * EVC_B_C, connecting User_B (Switch_3 interface 1) to User_C (Switch_3 interface 3).
+
+User_B's proxy port is PP (Switch_3 interface 4).
+
+::
+
+                                         PP
+                                         |
+                                         4
+                                         |
+           ┌────────┐   ┌────────┐   ┌────────┐
+ User_A -2-│Switch_1│-1-│Switch_2│-2-│Switch_3│-1- User_B
+           └────────┘   └────────┘   └────────┘
+                                          |
+                                          3
+                                          |
+                                        User_C
+
+For this example, consider `Switch_1`'s DPID: 00:00:00:00:00:00:00:01 and `Switch_3`'s DPID: 00:00:00:00:00:00:00:03.
+
+::
+
+ curl -s -X POST -H 'Content-type: application/json' http://localhost:8181/api/kytos/topology/v3/interfaces/00:00:00:00:00:00:00:01:2/metadata -d '{"proxy_port": 0}'
+ curl -s -X POST -H 'Content-type: application/json' http://localhost:8181/api/kytos/topology/v3/interfaces/00:00:00:00:00:00:00:03:3/metadata -d '{"proxy_port": 0}'
+ curl -s -X POST -H 'Content-type: application/json' http://localhost:8181/api/kytos/topology/v3/interfaces/00:00:00:00:00:00:00:03:1/metadata -d '{"proxy_port": 4}'
+
+
+With the configuration above, the EVCs will be become:
+
+  1. EVC_A_B:
+    1. User_A --> User_B: User_A (Switch_1 interface 2) to PP (Switch_3 interface 4) and PP (Switch_3 interface 4) to User_B (Switch_3 interface 1)
```

Comment:
To make sure I followed the idea correctly, is it also desirable to have traffic going through PP for this inter-switch EVC_A_B? I had the impression that only intra-switch EVCs should try to use the `proxy_port`. As it is, it seems that whenever a proxy port is configured if multiple inter-EVCs are configured in the same port they'll start using the same PP leading to potential bottlenecks depending on the traffic load? I wonder if `proxy_port` should be ignored for this case

Reply:
Proxy ports are mandatory for intra-switch EVCs and environments that mix intra- and inter-switch EVCs. If a proxy port is shared (1:N), then we will have the bottlenecks as per your comment. However, if a proxy port is dedicated, the proxy port will match the UNI's bandwidth. In this case, if there is a bottleneck at the proxy port, that bottleneck would exist for the UNI anyway.